data/sql_updates/reports_pac_party.sql

```diff
@@ -45,8 +45,8 @@ select
     non_alloc_fed_elect_actvy_per as non_allocated_fed_election_activity_period,
     non_alloc_fed_elect_actvy_ytd as non_allocated_fed_election_activity_ytd,
     shared_nonfed_op_exp_per as nonfed_share_allocated_disbursements_period,
-    ttl_op_exp_per as offsets_to_operating_expenditures_period,
-    ttl_op_exp_ytd as offsets_to_operating_expenditures_ytd,
+    offests_to_op_exp as offsets_to_operating_expenditures_period,
+    offsets_to_op_exp_ytd_i as offsets_to_operating_expenditures_ytd,
```

Comment:
yea, and it means that we have a problem in the totals, I documented in the issue.

The problem is that on the forms they ask the same question more than once, but people don't always fill out things twice, so you need to take the greatest of both values.

That was the way on the original table, then we switched to a case where that wasn't true, Then we switched to a case where that was true again, but did not add the greatest back in.